### PR TITLE
PA-2502 View Entire Province

### PR DESCRIPTION
**Pull request by agalbrai**
https://user-images.githubusercontent.com/33818575/105398379-27dfb280-5bd7-11eb-8af1-f63de71fd3fe.mp4

Also added a tool tip for the marker legend

### CODE REVIEW

File: frontend/src/components/maps/leaflet/ZoomOut/ZoomOutButton.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import Control from 'react-leaflet-control';
+import { Button } from 'react-bootstrap';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+import styled from 'styled-components';
+import { FaExpandArrowsAlt } from 'react-icons/fa';
+import { LatLngBounds, Map as LeafletMap } from 'leaflet';
+import { MapProps as LeafletMapProps, Map as ReactLeafletMap } from 'react-leaflet';
+
+const ZoomButton = styled(Button)`
```
Comment (jdtoombs) [inline frontend/src/components/maps/leaflet/ZoomOut/ZoomOutButton.tsx:10]: Woo styled components 👍

**Review by jdtoombs — APPROVED** [review]
Looks good!

File: frontend/src/components/maps/leaflet/Legend/LegendControl.tsx [hunk 2]
```diff
@@ -24,9 +25,11 @@ export const LegendControl: React.FC = () => {
   return (
     <Control position="topleft">
       <ClickAwayListener onClickAway={() => setVisible(false)}>
-        <LegendButton ref={target} onClick={() => setVisible(!visible)}>
-          <FiMapPin />
-        </LegendButton>
+        <TooltipWrapper toolTipId="marker-legendId" toolTip="Marker legend">
```
Comment (devinleighsmith) [inline frontend/src/components/maps/leaflet/Legend/LegendControl.tsx:28]: Thanks for this.

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/components/maps/leaflet/ZoomOut/ZoomOutButton.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  width: 40px;
+  height: 40px;
+`;
+
+export type ZoomOutProps = {
+  map: React.RefObject<ReactLeafletMap<LeafletMapProps, LeafletMap>>;
+  bounds: LatLngBounds;
+};
+
+export const ZoomOutButton: React.FC<ZoomOutProps> = ({ map, bounds }) => {
```
Comment (Fosol) [inline frontend/src/components/maps/leaflet/ZoomOut/ZoomOutButton.tsx:22]: Add docs

File: frontend/src/components/maps/leaflet/ZoomOut/ZoomOutButton.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import { MapProps as LeafletMapProps, Map as ReactLeafletMap } from 'react-leaflet';
+
+const ZoomButton = styled(Button)`
+  background-color: #ffffff !important;
+  color: #121212 !important;
+  width: 40px;
+  height: 40px;
+`;
+
+export type ZoomOutProps = {
```
Comment (Fosol) [inline frontend/src/components/maps/leaflet/ZoomOut/ZoomOutButton.tsx:17]: Add docs

**Review by Fosol — APPROVED** [review]

**Pull request merged**